### DISCUSSION TITLE
Ability to set state to 'empty' on Bulk Edit

### DISCUSSION THREAD

**Discussion by emmapeel2** [WeblateOrg/weblate#11727] [Ideas] [answered]
I need an easy way to remove erroneous translations while mantaining our project.

It would be great to be able to set state to Empty for example in https://hosted.weblate.org/projects/tor/tor-browser/tb-android/#bulk-edit

I think this power should be only for mantainers of the translations, not needed for translators or reviewers.

In my case, my translation memory is contaminated with source strings, and I want to make empty all the strings that have check:same.

**Comment by nijel** [comment, marked as answer]
Tracked as https://github.com/WeblateOrg/weblate/issues/11887
